firstvoices/backend/tests/test_apis/test_story_api.py

```diff
@@ -255,3 +256,67 @@ def test_pages_order(self):
         response_data = json.loads(response.content)
         assert response_data["pages"][0]["text"] == page1.text
         assert response_data["pages"][1]["text"] == page2.text
+
+    @pytest.mark.django_db
+    def test_update_page_order(self):
+        site = factories.SiteFactory.create(visibility=Visibility.PUBLIC)
+
+        user = factories.get_non_member_user()
+        factories.MembershipFactory.create(user=user, site=site, role=Role.ASSISTANT)
+        self.client.force_authenticate(user=user)
+
+        story = factories.StoryFactory.create(visibility=Visibility.TEAM, site=site)
+
+        page1 = factories.StoryPageFactory.create(story=story, ordering=0)
+        page2 = factories.StoryPageFactory.create(story=story, ordering=1)
+
+        assert Story.objects.filter(site=site).count() == 1
+        assert StoryPage.objects.all().count() == 2
+        assert StoryPage.objects.filter(story=story).count() == 2
+        assert StoryPage.objects.get(id=page1.id).ordering == 0
+        assert StoryPage.objects.get(id=page2.id).ordering == 1
+
+        response = self.client.get(
+            self.get_detail_endpoint(key=story.id, site_slug=site.slug)
+        )
+
+        assert response.status_code == 200
+        response_data = json.loads(response.content)
+        assert response_data["pages"][0]["text"] == page1.text
+        assert response_data["pages"][1]["text"] == page2.text
+
+        data = {
+            "title": story.title,
+            "visibility": story.get_visibility_display(),
+            "author": "",
+            "title_translation": "",
+            "introduction": "",
+            "introduction_translation": "",
+            "notes": [],
+            "pages": [str(page2.id), str(page1.id)],
+            "acknowledgements": [],
+            "hide_overlay": False,
+            "exclude_from_games": False,
+            "exclude_from_kids": False,
+            "related_audio": [],
+            "related_images": [],
+            "related_videos": [],
+        }
+
+        response = self.client.put(
```

Comment:
Please test the patch for this-- it's what will be used 99% of the time for page order updates